streampark-console/streampark-console-newui/src/views/system/variable/components/VariableDrawer.vue

```diff
@@ -114,6 +115,17 @@
         componentProps: { rows: 4 },
         rules: [{ max: 100, message: t('system.variable.form.descriptionMessage') }],
       },
+      {
+        field: 'desensitization',
+        label: 'Desensitization',
+        component: 'Switch',
+        componentProps: {
+          checkedChildren: 'ON',
+          unCheckedChildren: 'OFF',
+        },
+        defaultValue: false,
+        afterItem: h('span', { class: 'conf-switch' }, 'If set to ON, the value of the variable will be displayed as ******** elsewhere.'),
```

Comment:
Whether desensitization is required, e.g: desensitization of sensitive data such as passwords, if enable variable value will be displayed as ********